masterlist.yaml

```diff
@@ -28,10 +28,84 @@ globals:
         text: '[Postagem mais recente do LOOT](%1%).'
     subs: [ 'https://loot.github.io/latest-thread/' ]
 
-groups: []
+groups:
+- name: &dlcGroup DLC
+
+- name: &unofficialPatches
+  after:
+  - *dlcGroup
+
+- name: &otherMods Other Mods
```

Comment:
This should probably be the `default` group, instead of `Other Mods`, so that dynamic patches load after everything that doesn't explicitly have another group defined.